dev_tools/utils/nxdl_utils.py

```diff
@@ -783,17 +759,15 @@ def walk_elist(elist, html_name):
                     if fitting_child is not None:
                         child = fitting_child
                     break
-        elist[ind] = child
```

Comment:
here a functional difference again over several lines

Reply:
indeed, it is now fixed